README.md

```diff
@@ -186,6 +188,8 @@ source /usr/local/share/chruby/chruby.sh
 source /usr/local/share/chruby/auto.sh
 ```
 
+*Note:* OSX does not automatically execute `~/.bashrc`, instead try adding to `~/.profile`.
+
```

Comment:
Should reference `/etc/bashrc` and not `~/.profile` since this is for System Wide configuration. I really wish OS X supported `~/.bashrc` and `/etc/profile.d/`.